.buildkite/test-sorbet-rubyfmt.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
```

Comment:
I think that the buildkite yaml pipeline syntax allows setting environment variables. Do you think we could set an environment variable and read that inside the existing `test-static-sanitized.sh` file so that we don't have to duplicate so much code?

Reply:
Seems reasonable to me!